locksmith/README.md

```diff
@@ -0,0 +1,84 @@
+# Locksmith
+
+Locksmith provides the ability to store metadata associated with Locks outside of the Ethereum blockchain, with reasonable permission around writes.
+
+In the current iteration, reads are publically readable.
+
+## Getting Started
+
+Locksmith currently utilizes [Postgres](https://www.postgresql.org/) as its datastore and will require access to a running Instance. 
+
+You can configure your connection details in a few ways, here are the suggested methods:
+
+1. Configure environment variables (Locksmith with recogize these placed in a .env file at the root of the application)
+   * **DB_USERNAME** - Database User
+   * **DB_PASSWORD** - Password of User
+   * **DB_NAME** - Database Name
+   * **DB_HOSTNAME** - Database Host
+
+
+
+
+   __Note__: The following can be used to quickly get started setting up your __local development__ database.
+   ```sql
+   psql -c 'create database {DATABASE NAME};' -U postgres
+   psql -c "create user {DATASBASE USER NAME} with password '{PASSWORD}';" -U postgres
+   ```
+
+2. If your configuration needs are more demanding than this, further configuration can be established via the storage configuration file  `config\config.js`
+
+Once configured, you will be able to start the application. `npm run dev` or `npm start` will suffice depending on your needs for running Locksmith
+
+### Persistance Information
+
+General database administration is out of scope for this document; it is generally advised to provide the database user associated with the application with only the permissions required.
+
+In development mode, migrations will run upon start up ensuring that your persistance layer is up to date. In production, migrations will need to be performed manually. This can be performed via `npm db:migrate`. Please review the migrations included so that you are aware of the items included.
```

Comment:
```suggestion
In development mode, migrations will run upon start up ensuring that your persistence layer is
up to date. In production, migrations will need to be performed manually. This can be performed
via `npm db:migrate`. Please review the migrations included so that you are aware of the items included.
```